packages/slice-machine/src/modules/selectedCustomType/sagas.ts

```diff
@@ -34,7 +34,12 @@ export function* saveCustomTypeSaga() {
     yield put(saveCustomTypeCreator.success({ customType: currentCustomType }));
     yield put(
       openToasterCreator({
-        content: "Model has been generated successfully!",
+        content: `${
```

Comment:
We have config btw for that in "customTypesMessages.ts", we want to centralize these kinds of things. You can see others examples for this.

Reply:
done